SurveyApp.DAL/EntityModels/AnswerDataModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace SurveyApp.DAL.EntityModels
+{
+    public class AnswerDataModel
```

Comment:
"DataModel' suffix seems redundant. 'AnswerDataModel' table looks strange in database. 'Answer' is better naming for the model and table. To distinguish Answer model in data Access and Answer model in Business layer we can use namespaces.